cmd/tvx/extract_message.go

```diff
@@ -43,6 +42,15 @@ func doExtractMessage(opts extractOpts) error {
 		return fmt.Errorf("failed to resolve message and tipsets from chain: %w", err)
 	}
 
+	// Assumes that the desired message isn't at the boundary of network versions.
+	// Otherwise this will be inaccurate. But it's such a tiny edge case that
```

Comment:
These are actually a pretty key edge case that we need to be testing, esp what happens around null tipsets...but that's for later...